File: classmate/QuizApp/app/src/main/java/com/hikmatillo/quizapp/StartActivity.java
package com.hikmatillo.quizapp;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.TextView;

import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;


public class StartActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_start);


        Button startBtn = findViewById(R.id.StartButton);

        startBtn.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(StartActivity.this, GameActivity.class);
                startActivity(intent);


            }
        });
    }
}

File: classmate/WordGane/app/src/main/java/com/example/wordgane/ui/StartActivity.kt
package com.example.wordgane.ui

import android.annotation.SuppressLint
import android.content.Intent
import android.os.Bundle
import com.example.wordgane.R
import com.example.wordgane.core.BaseActivity
import com.example.wordgane.core.cashe.AppCashe
import com.example.wordgane.databinding.ActivityStartBinding

class StartActivity : BaseActivity() {

    private lateinit var binding: ActivityStartBinding

    @SuppressLint("UseCompatLoadingForDrawables")
    override fun setTheme() {

        when (AppCashe.getObject().getTheme()) {
            0 -> {
                binding.DefSph.setImageDrawable(getDrawable(R.drawable.back_photo2))
            }

            1 -> {
                binding.DefSph.setImageDrawable(getDrawable(R.drawable.back_photo3))

            }

            2 -> {
                binding.DefSph.setImageDrawable(getDrawable(R.drawable.back_photo4))
            }
        }
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        binding = ActivityStartBinding.inflate(layoutInflater)

        setContentView(binding.root)

        loadView()


    }

    private fun loadView() {


        binding.startButton.setOnClickListener {
            val intent = Intent(this, MainActivity::class.java)
            startActivity(intent)
        }

        binding.settingButton.setOnClickListener {

            val intent = Intent(this, SettingActivity::class.java)
            startActivity(intent)
        }

    }


}